koku/masu/external/accounts/db/cur_accounts_db.py

```diff
@@ -22,7 +22,26 @@
 
 # pylint: disable=too-few-public-methods
 class CURAccountsDB(CURAccountsInterface):
-    """Provider interface defnition."""
+    """Provider interface definition."""
+
+    @staticmethod
+    def get_authentication(provider):
+        """Return either provider_resource_name or credentials."""
+        if provider.authentication.provider_resource_name:
```

Comment:
For tracking purposes, it might be good to include a comment or extra info in the docstring. This is only conditional because the `provider_resource_name` is deprecated. That will make it easier to find these sections of code when we go to remove the deprecated bits.